### ISSUE TITLE
Surface loss Keras

### ISSUE THREAD

**Issue by AmericaBG** [LIVIAETS/boundary-loss#27]
Hi!

I've a question about the surface-loss Keras.

When I do:  model.compile(optimizer=opt, loss= ...)

In the 'loss' parameter, should I call only to surface-loss? Or sourface-loss+some loss function?

Thank you very much for your attention!

**Comment by HKervadec** [comment]
Hi,
Could this be what you are looking for ? https://github.com/LIVIAETS/boundary-loss/pull/28

```python
def gl_sl_wrapper(alpha):
    def gl_sl(y_true, y_pred):
        return alpha * generalized_dice_loss(
            y_true, y_pred) + (1 - alpha) * surface_loss_keras(y_true, y_pred)
    return gl_sl

model.compile(loss=gl_sl_wrapper(alpha))
```
